### PR TITLE
Add ESET scanning script

### PR DESCRIPTION
**Pull request by joeldeteves**
This PR adds a script to scan using ESET Endpoint Antivirus (also should work with ESET Endpoint Security, etc).

See https://help.eset.com/efsw/10/en-US/idh_config_ermm.html for more info

### CODE REVIEW

**Comment by silversword411** [comment]
Want to add syntax too?

**Review by silversword411 — COMMENTED** [review]
Kinda after the fact now, but json needs 2 \'s for escaping apparently

File: community_scripts.json [hunk 1]
```diff
@@ -1622,5 +1622,20 @@
...
     "category": "TRMM (Win):Monitoring"
+  },
+  {
+    "guid": "79108c30-8010-4303-81c0-6b56623a3b35",
+    "filename": "Win_ESET_ThreatScan.ps1",
+    "submittedBy": "https://github.com/lcsnetworks",
+    "name": "ESET - Threat Scan",
+    "description": "Runs a Smart Scan on the target system using ESET Endpoint Antivirus. Can be adjusted to different drives / paths + scan profiles via script Arguments.",
+    "args": [
+      "-target C:\\",
```
[the remark this answers is not in the document]
Comment (joeldeteves) [reply community_scripts.json:1634]: Yes, `\` is an escape character in JSON. Therefore, in order to include a `\` in the parameter, there needs to be two of them.

[the remark this answers is not in the document]
Comment (silversword411) [reply community_scripts.json:1634]: Invisible comments solved. Yay!

**Pull request merged**